src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -674,6 +674,16 @@ void MatterGeneralCommissioningPluginServerInitCallback()
     Server::GetInstance().GetFabricTable().AddFabricDelegate(&fabricDelegate);
 }
 
+void MatterGeneralCommissioningPluginServerShutdownCallback()
+{
+    static GeneralCommissioningFabricTableDelegate fabricDelegate;
```

Comment:
This makes two definitions of static `fabricDelegate`, can we move this out?

Reply:
Done